DuckDuckGo/Localizable.xcstrings

```diff
@@ -40563,6 +40563,9 @@
           }
         }
       }
+    },
+    "Please give us your feedback:" : {
+
     },
```

Comment:
Silly question, is it intended that these strings are in this format? I see from the others that they typically have the key at the top level, then comment etc. underneath it.

Reply:
That was auto-appended to the xcstrings file. Removed.